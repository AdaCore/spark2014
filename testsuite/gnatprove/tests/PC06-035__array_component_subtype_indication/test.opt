ALL XFAIL missing check on subtype indication of array component type