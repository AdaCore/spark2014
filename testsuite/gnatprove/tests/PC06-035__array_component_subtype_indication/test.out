array_component.ads:4:04: medium: range check might fail (e.g. when D = 0)
